studio/components/OpenSharedButton.tsx

```diff
@@ -0,0 +1,9 @@
+import { Button } from "@sanity/ui";
```

Comment:
for a11y this should be a link and not a button. This is kinda my fault for saying button when describing the task 😅

Reply:
when adding a link, I don't think we want to open a new window or tab, but stay in the same window. 🤔

Reply:
Are you sure? I’m just thinking it could really annoying that if you’re editing a blogpost and then have to navigate back if you don't want to add anything to shared.

Reply:
But i guess this is not a big issue as Sanity autosaves everything